backends/memcheck/ceed-memcheck-qfunction.c

```diff
@@ -63,8 +63,12 @@ static int CeedQFunctionApply_Memcheck(CeedQFunction qf, CeedInt Q, CeedVector *
     // Note: need field size because vector may be longer than needed for output
     CeedCallBackend(CeedQFunctionFieldGetSize(output_fields[i], &field_size));
     for (CeedSize j = 0; j < Q * field_size; j++) {
-      CeedCheck(!isnan(impl->outputs[i][j]), ceed, CEED_ERROR_BACKEND, "QFunction output %d entry %ld is NaN after restoring write-only access", i,
-                j);
+      char *kernel_path, *kernel_name;
+      CeedCallBackend(CeedQFunctionGetSourcePath(qf, &kernel_path));
+      CeedCallBackend(CeedQFunctionGetKernelName(qf, &kernel_name));
```

Comment:
This should be outside of the loop

Reply:
Jinx. I couldn't decide where to place them so I just threw them there.

Reply:
Ha, You could do 

```
  {
    char *foo, *bar;

    // Get the ptrs
    for (....) {
    }
  }
```

Reply:
Pushed up. I put it outside of both for loops